guides/functor-combinator-instances-and-class1s.md

```diff
@@ -0,0 +1,83 @@
+# What
+
+CLC has approved the [proposal #10](https://github.com/haskell/core-libraries-committee/issues/10), to relax instances for Functor combinators and put superclasses on \<class>1 to make the change less-breaking.
+
+Before, e.g.:
+```haskell
+instance (Eq1 f, Eq1 g, Eq a) => Eq (Compose f g a)
+
+class Eq1 f
+
+class Eq2 f
+```
+
+After, e.g.:
+```haskell
+instance Eq (f (g a)) => Eq (Compose f g a)
+
+class (forall a. Eq a => Eq (f a)) => Eq1 f
+
+class (forall a. Eq a => Eq1 (f a)) => Eq2 f
+```
+
+# When
+
+At the very earliest, the change may appear in GHC 9.6 (~ Q1 2023).
+
+# How
+
+This is a breaking change:
+
+1. People could define e.g. `Eq1` without `Eq`, or `Eq2` without `Eq1`
+
+2. People could write complicated instances on various abstract machinary that doesn't abide by the new superclasses.
+
+However, the breakage is of limited scope.
+An impact analysis (please see [comments following here](https://github.com/haskell/core-libraries-committee/issues/10#issuecomment-1166669613))
+showed that a only handle of packages that sucessfully pass dependency resolution on Stackage `nightly-2022-06-17` break with this change.
+Patches have been submitted to each of them.
+
+The migration policy for this change is backward-compatible: you can migrate already and still retain compatibility with existing GHCs.
+Because of this, CLC suggests applying patches at your earliest convenience.
+
+1. Add any missing instances
```

Comment:
Let's spell this out more explicitly "If you defined `instance Eq1 f` but omitted `instance Eq a => Eq (f a)`, you must now define the latter, e. g., as `(==) = eq1`. If you defined `instance Eq2` f ..."

Please also link relevant PRs from https://github.com/haskell/core-libraries-committee/issues/10#issuecomment-1166669613 as examples.